service/src/test/java/cz/muni/fi/pa165/BeanMappingServiceTest.java

```diff
@@ -0,0 +1,95 @@
+package cz.muni.fi.pa165;
+
+import cz.muni.fi.pa165.dao.PersonDAO;
+import cz.muni.fi.pa165.dto.PersonDTO;
+import cz.muni.fi.pa165.entity.Person;
+import cz.muni.fi.pa165.enums.PersonRole;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+
+/**
+ * @author mhajas
+ */
+public class BeanMappingServiceTest extends AbstractServiceTest {
+
+
+    @Test
+    public void objectMapping() {
+        Person person = new Person();
+        PersonDTO personDTO = new PersonDTO();
+
+        person.setName("Michal");
```

Comment:
To uz by som to radsej hodil vsetko do Before metody, ale prislo mi to zbytocne, ked ide len o tieto dva testy. Ale spravim to, aspon pre tie, ktore sa tam vytvaraju duplicitne.